.management/branch_names.py

```diff
@@ -1,30 +1,85 @@
-import random
 import os
+import random
+
+import git
+
+SET_LENGTH = 10
+RANDOM_WORDS_PER_NAME = 3
+MINIMUM_WORDS_LENGTH = 2
+MAXIMUM_WORDS_LENGTH = 9
 
 
 def main():
     print("I'm the branch name generator.\nHere are some names for you:\n")
-    # Think about changing this to movie names from imdb database
     words = load_words()
     print(len(words))
 
     # Filter to mostly normal pronounceable words
-    words = set(filter(lambda x: 2 < len(x) < 9, words))
+    words = set(filter(lambda x: MINIMUM_WORDS_LENGTH < len(x) < MAXIMUM_WORDS_LENGTH, words))
```

Comment:
This range ought to be params (with argparse)

Reply:
Done